### PR TITLE
dup_timed_event() wasn't duping the disabled bit

### PR DESCRIPTION
**Pull request by bhroam**
#### Describe Bug or Feature
The function dup_timed_event() wasn't copying over the disabled bit.  This means when the universe was dup'd, the disabled events would be performed.  This could cause top jobs to be scheduled too soon.  This really only affected preemption with sched_preempt_enforce_resumption on.  This was the only place timed events would be marked disabled and then the universe was dup'd.  It would only affect the cycle jobs are preempted.  Future cycles would be fine.

#### Describe Your Change
In dup_timed_event(), copy over the disabled bit.

#### Attach Test and Valgrind Logs/Output
There really isn't any bang for our buck for writing a test.  It only affects a small corner case, and the outcome wasn't that bad.

### CODE REVIEW

File: src/scheduler/simulate.c [hunk 1]
```diff
@@ -1099,6 +1099,7 @@ dup_timed_event(timed_event *ote, server_info *nsinfo)
 		return NULL;
 
 	nte = create_event(ote->event_type, ote->event_time, event_ptr, ote->event_func, ote->event_func_arg);
+	set_timed_event_disabled(nte, ote->disabled);
```
Comment (agrawalravi90) [inline src/scheduler/simulate.c]: why not just do "nte->disabled = ote->disabled" ?

Comment (bhroam) [reply src/scheduler/simulate.c]: If there is a setter, why not use it?  It means if I ever need to do something every time the disabled bit is set, I can do it there and trust it will always happen when an event is disabled.

Comment (vstumpf) [reply src/scheduler/simulate.c]: But the event should just be duped, right? So if we act on it, it might no longer be a dupe.

Comment (bhroam) [reply src/scheduler/simulate.c]: A good example is the call to create_event().  It handles setting the run_event and end_event pointers to a resource_resv structure.  This is something that needs to be done every time we create an event.  The setter should do everything to disable an event.  If an event is disabled in the old universe, it should be disabled in the new universe.  I don't understand why we shouldn't call it.

Comment (agrawalravi90) [reply src/scheduler/simulate.c]: > if there is a setter, why not use it?

Since the setter doesn't do anything other than assignment, it seemed unnecessary. If you fore-see us adding more code in the future, then cool. The -O2 flag should inline this function anyways, so there shouldn't be any overhead as such.
